logwarn_openqa

```diff
@@ -69,7 +69,7 @@ $logwarn ${options} -m '\[.*:?(debug|info|warn|error)\]' -p ${file} \
     `# https://progress.opensuse.org/issues/53474` \
     '!error\].*File exists at.*Schema/Result/Jobs' \
     `# https://progress.opensuse.org/issues/105909` \
-    '!warn\].*Ignoring invalid group \{"name":"123"\} when creating new job ' \
+    '!warn\].*Ignoring invalid group \{"name":"[0-9]+"\} when creating new job ' \
```

Comment:
Isn't it just coincidence that the name only contains digits here?

Reply:
Judging from the code it is definitely possible to get non-numeric log entries. ("name" is supposed to be the group's name.) Obviously this whole error should rather be propagated to the user and has no use in our own logs but I suppose that's a different issue…

Reply:
well, that's exactly the point of the referenced issue https://progress.opensuse.org/issues/105909 . Here we *only* handle alarm reporting, meaning as soon as we decided that we found a non-critical issue we handle it in a ticket and don't alarm on it repeatedly

Reply:
Putting a meaningful regex here is IMO part of handling the alarm reporting. (And propagating the error the user is not.)

Reply:
I agree